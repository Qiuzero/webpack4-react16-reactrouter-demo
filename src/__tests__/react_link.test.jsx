import React from 'react';
import renderer from 'react-test-renderer';
import ALink from '../components/Link.react';

it('正确的渲染', () => {
  const tree = renderer
    .create(<ALink page="https://www.walkerfree.com">Walkerfree</ALink>)
    .toJSON(); expect(tree).toMatchSnapshot();
});
