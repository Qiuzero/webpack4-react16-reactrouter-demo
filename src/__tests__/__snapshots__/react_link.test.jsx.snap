// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`正确的渲染 1`] = `
<a
  className="normal"
  href="https://www.walkerfree.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Walkerfree
</a>
`;
